packages/playwright/test/tsTest.ts

```diff
@@ -0,0 +1,21 @@
+// This file isn't executed; we only care that typescript can successfully
+// build/typecheck. It is a smoke test that our build process is producing a
+// basically-reasonable index.d.ts.
+
+// Detailed tests of individual types should instead be covered by unit tests.
+
+import { AxeBuilder as NamedImportAxeBuilder } from '../dist/index.js';
+import DefaultImportAxeBuilder from '../dist/index.js';
```

Comment:
Could probably do it like this too. Two imports on the same file is a little odd.

```suggestion
import {
  default as DefaultImportAxeBuilder,
  AxeBuilder as NamedImportAxeBuilder
} from '../dist/index.js';
```